core/offchain.go

```diff
@@ -213,6 +193,29 @@ func (bc *BlockChain) CommitOffChainData(
 		}
 	}
 
+	// Update voting power of validators for all shards
+	tempValidatorStats := map[common.Address]*staking.ValidatorStats{}
```

Comment:
Why did this block move?

Reply:
just to keep it closer to the two places where the stats are used.